File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0208Q01GrdOCS0208Info.java
package nta.med.data.model.ihis.ocsa;

public class OCS0208Q01GrdOCS0208Info {
	private String gubun ;
	private String bogyongCode ;
	private String bogyongName ;
	private String userYn ;
	private String bogyongGubun ;
	private Double seq ;
	public OCS0208Q01GrdOCS0208Info(String gubun, String bogyongCode,
			String bogyongName, String userYn, String bogyongGubun, Double seq) {
		super();
		this.gubun = gubun;
		this.bogyongCode = bogyongCode;
		this.bogyongName = bogyongName;
		this.userYn = userYn;
		this.bogyongGubun = bogyongGubun;
		this.seq = seq;
	}
	public String getGubun() {
		return gubun;
	}
	public void setGubun(String gubun) {
		this.gubun = gubun;
	}
	public String getBogyongCode() {
		return bogyongCode;
	}
	public void setBogyongCode(String bogyongCode) {
		this.bogyongCode = bogyongCode;
	}
	public String getBogyongName() {
		return bogyongName;
	}
	public void setBogyongName(String bogyongName) {
		this.bogyongName = bogyongName;
	}
	public String getUserYn() {
		return userYn;
	}
	public void setUserYn(String userYn) {
		this.userYn = userYn;
	}
	public String getBogyongGubun() {
		return bogyongGubun;
	}
	public void setBogyongGubun(String bogyongGubun) {
		this.bogyongGubun = bogyongGubun;
	}
	public Double getSeq() {
		return seq;
	}
	public void setSeq(Double seq) {
		this.seq = seq;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0208Q01GrdOCS0208Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS0208Q01GrdOCS0208Info
	{
		private String _gubun;
		private String _bogyongCode;
		private String _bogyongName;
		private String _userYn;
		private String _bogyongGubun;
		private String _seq;

		public String Gubun
		{
			get { return this._gubun; }
			set { this._gubun = value; }
		}

		public String BogyongCode
		{
			get { return this._bogyongCode; }
			set { this._bogyongCode = value; }
		}

		public String BogyongName
		{
			get { return this._bogyongName; }
			set { this._bogyongName = value; }
		}

		public String UserYn
		{
			get { return this._userYn; }
			set { this._userYn = value; }
		}

		public String BogyongGubun
		{
			get { return this._bogyongGubun; }
			set { this._bogyongGubun = value; }
		}

		public String Seq
		{
			get { return this._seq; }
			set { this._seq = value; }
		}

		public OCS0208Q01GrdOCS0208Info() { }

		public OCS0208Q01GrdOCS0208Info(String gubun, String bogyongCode, String bogyongName, String userYn, String bogyongGubun, String seq)
		{
			this._gubun = gubun;
			this._bogyongCode = bogyongCode;
			this._bogyongName = bogyongName;
			this._userYn = userYn;
			this._bogyongGubun = bogyongGubun;
			this._seq = seq;
		}

	}
}